### ISSUE TITLE
SPDY support

### ISSUE THREAD

**Issue by kazuho** [h2o/h2o#121]
Until all the web browsers start supporting HTTP/2 it might be worthwhile to support the SPDY protocol.

It would not be so difficult to implement the protocol handler as we can use [spdylay](https://github.com/tatsuhiro-t/spdylay).

**Comment by mingodad** [comment]
A thumbs up, that will make h2o a lot more worth !

**Comment by kazuho** [comment]
I am afraid I cannot do it myself, but I would appreciate it if someone could work on the issue.

**Comment by szepeviktor** [comment]
Until then it is a promising experiment.

**Comment by swinghu** [comment]
:+1:

**Comment by lkwg82** [comment]
this is outdated!

**Comment by kazuho** [comment]
Agreed.  There is no need to support SPDY, clients are actually dropping support for the protocol in favor of HTTP/2.

**Comment by lkwg82** [comment]
... and close ;)
